setup_robotino/robotino-adduser

```diff
@@ -34,7 +34,7 @@ if [ "$1" != "" ]; then
 	sudo -u $1 cat /u/$1/.ssh/id_rsa.pub | ssh $1@robotino "cat >> /u/$1/.ssh/authorized_keys"
```

Comment:
i would replace all the cat into authorized_keys with the standard command for this which is ssh-copy-id

Reply:
The rsync command fails for us when trying to rely on the public key authentication. Was this supposed to be used with a root password?

Reply:
did you call the script with sudo or without sudo? 
try from scrat pc:
```
su root
ssh robotino
```

is this working??

Reply:
This is not working without a password. The only entry in authorized_keys of root is from squirrel@scrat. So without the password this will always fail.

Reply:
su root
ssh-copy-id root@robotino